internal/cmd/utils.go

```diff
@@ -257,6 +257,18 @@ func dbNameArg(cmd *cobra.Command, args []string, toComplete string) ([]string,
 	return []string{}, cobra.ShellCompDirectiveNoFileComp
 }
 
+func dbNameAndOrgArgs(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+	client, err := createTursoClientFromAccessToken(false)
+	if err != nil {
+		return nil, cobra.ShellCompDirectiveNoFileComp | cobra.ShellCompDirectiveNoSpace
+	}
+	if len(args) == 1 {
+		orgs, _ := client.Organizations.List()
```

Comment:
This has the ability to spam our API hard.
Making an HTTP request to offer a completion can also make the CLI feel unresponsive.

Please open an issue to cache this request, similarly to what we do with the databases list.